tools/syscount.py

```diff
@@ -362,7 +362,7 @@ def parse_syscall(line):
     # Skip the first line, which is a header. The rest of the lines are simply
     # SYSCALL_NUM\tSYSCALL_NAME pairs.
     out = subprocess.check_output('ausyscall --dump | tail -n +2', shell=True)
-    syscalls = dict(map(parse_syscall, out.strip().split('\n')))
+    syscalls = dict(map(parse_syscall, out.decode().strip().split('\n')))
```

Comment:
We should be avoiding decode(). Does split(b'\n') work instead?

Reply:
split(b'\n') works but every syscall in the output will be prepended a 'b'. Just feel it prettier without 'b'.

Reply:
Can the tool use instead the printb function that was added to bcc.utils to handle this type of case? It directly uses `stdout.write()` instead of `print`, which will preserve whichever special characters were there in the process name.

Reply:
Ok, I'll send a v2 to use printb.